pkgi_vita.cpp

```diff
@@ -622,18 +623,33 @@ int pkgi_battery_is_charging()
     return scePowerIsBatteryCharging();
 }
 
-uint64_t pkgi_get_free_space(void)
+const char* pkgi_get_partition(void)
 {
+    Config config = pkgi_load_config();
+    const char* partition = config.install_psp_psx_location.c_str();
+    if ((partition == NULL) || (partition[0] == '\0')) {
```

Comment:
partition can't be null. Don't call c_str() above and use [std::string::empty](http://en.cppreference.com/w/cpp/string/basic_string/empty) for the if